tajo-plan/src/main/java/org/apache/tajo/plan/logical/AlterTableNode.java

```diff
@@ -42,6 +43,12 @@
   private KeyValueSet properties = new KeyValueSet();
   @Expose
   private AlterTableOpType alterTableOpType;
+  @Expose
+  private String[] columnNames;
```

Comment:
```colunmNames``` seems to mean partition columns, right? How about changing the name to be more intuitive?